src/org/labkey/test/components/ui/search/FilterExpressionPanel.java

```diff
@@ -5,15 +5,21 @@
 import org.labkey.test.components.Component;
 import org.labkey.test.components.WebDriverComponent;
 import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.RadioButton;
 import org.labkey.test.components.react.ReactSelect;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.ui.ExpectedConditions;
 
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
 import java.util.Map;
 
 public class FilterExpressionPanel extends WebDriverComponent<FilterExpressionPanel.ElementCache>
 {
+    private static final DateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd H:m");
```

Comment:
At some point in the far flung future we may want to consider moving away from having hard coded date formats.
The reason I mention this is because we have had a few recent bug fixes where non-US date format was not getting parsed correctly.